### ISSUE TITLE
the main window of the GUI is disappeared.

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1672]
_From [reza.ahm...@gmail.com](https://code.google.com/u/115909755058302275323/) on 2013-12-08T17:15:41Z_

Please note that reporting bugs from Spyder's help menu ("?" > "Report
issue...") will fill version details for Spyder, Python and Qt in this description for you. 

What steps will reproduce the problem?
1. running spyder.bat
   1. the green "Spyder 2" appears
   2. the main windows is invisible. What is the expected output? What do you see instead? After the green "Spyder 2" screen disappears the main screen is invisible. Actually the icon is active and the process is there, but the window is kind of invisible. What version of the product are you using? On what operating system? Spyder 2.2.5 - win32 - Python 2.7 - Windows 7 - 64bits 

Please provide any additional information below

. It was working recently very well, but suddenly the main window doesn't appear at all.

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1672_

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2013-12-11T17:27:57Z_

What happens if you open a terminal (i.e. cmd.exe), then run in it

spyder --reset

then start Spyder again?

You may also need to install pywin32 (if you don't have it installed).

**Comment by spyder-issues-migrator** [comment]
_From [reza.ahm...@gmail.com](https://code.google.com/u/115909755058302275323/) on 2013-12-12T02:44:16Z_

the spyder --reset apparently solved the problem. the output is:

C:\Users\reza\Desktop>spyder --reset
*** Reset Spyder settings to defaults ***
removing: C:\Users\reza\.spyder2\.inspector
removing: C:\Users\reza\.spyder2\.onlinehelp
removing: C:\Users\reza\.spyder2\.spyder.ini
removing: C:\Users\reza\.spyder2\.temp.py
removing: C:\Users\reza\.spyder2\template.py
removing: C:\Users\reza\.spyder2\.history.py
removing: C:\Users\reza\.spyder2\.history_internal.py
removing: C:\Users\reza\.spyder2\.ropeproject
removing: C:\Users\reza\.spyder2\.workingdir


thanks

**Comment by spyder-issues-migrator** [comment]
_From [stonebi...@gmail.com](https://code.google.com/u/116068254323340012360/) on 2013-12-12T11:03:43Z_

So it may be considered fixed by the usual "spyder --reset" workaround.

At school :
- this needs to be used about one time every 15 sessions on slow and very-poor in memory PC,
- nearly never on recent PCs.

**Comment by spyder-issues-migrator** [comment]
_From [contrebasse](https://code.google.com/u/contrebasse/) on 2013-12-12T11:59:49Z_

It seems that the config file is easily corrupted as many issues are fixed by resetting the config. Maybe the file saving should be refactored somehow sometimes in the future ? If a very resilient interface is wanted, sqlite can be used as the db is "guaranteed" to never be corrupted even if the application crashes in the middle of the file writing.

**Comment by ccordoba12** [comment]
Closing this old bug.

**Comment by kannanchandraskaran** [comment]
I had the same issue that the GUI disappearing. then ran "spyder --reset" in command prompt (cme.exe).

**Comment by jmoraispk** [comment]
This still happens at times. 
The problem seems related with changing screens. I closed spyder using one monitor that was virtually above my main screen and then connected a different monitor that was virtually at the left of my main screen, I noticed the spyder window going to some place above my main monitor. 

The proposed fix of resetting Spyder still works though. BUT it would be nice to reset/delete _only_ the relevant file - some configurations and all shortcuts went away with this fix.
